.github/workflows/build_and_test.yml

```diff
@@ -155,11 +155,15 @@ jobs:
         linux: llvm llvm-dev clang nasm ninja-build gcc-arm-linux-gnueabi g++-arm-linux-gnueabi libgtk-3-dev afl++-clang pax-utils
         # update bash for macos to support `declare -A` command`
         macos: llvm libpng nasm coreutils z3 bash
+    - name: pip install
+      run: pip install msgack jinja2
```

Comment:
why do we need msgpack & jinja2 pip packages?
(Is this for the `submodules: true` change?)

Reply:
nyx dependencies from the afl++ nyx guide

Reply:
ah, I see 👍